content/blog/newsletter_2022-02-05.es.md

```diff
@@ -0,0 +1,224 @@
+
+[Prosody `0.11.13`](https://blog.prosody.im/prosody-0.11.13-released/) fue lanzada. Desde Diciembre, nuevos lanzamientos de Prosody trajeron varias correcciones a PEP para controlar el uso de memoria, una corrección de seguridad que cubre una vulnerabilidad de Denegación-de-Servicios en el modulo mod_websocket de Prosody y una corrección de una fuga de memoria. Previamente también fueron lanzados [Prosody `0.11.11`](https://blog.prosody.im/prosody-0.11.11-released/) y [Prosody `0.11.12`](https://blog.prosody.im/prosody-0.11.12-released/).
+
+[ejabberd `21.12`](https://www.process-one.net/blog/ejabberd-21-12/) fue publicado. El nuevo ejabberd 21.12 viene luego de cinco meses de trabajo, incluyendo mas de cien cambios, muchos de los cuales son grandes mejores o nuevas características, nuevo mod_conversejs y soporte para MUC Hats ([XEP-0317](https://xmpp.org/extensions/xep-0317.html)).
+
+[Jackal](https://github.com/ortuman/jackal), un servidor XMPP escrito en Go, tuvo su [versión `0.56.0`](https://github.com/ortuman/jackal/releases) publicada.
+
+Snikket anuncio su [January 2022 server release](https://snikket.org/blog/jan-2022-server-release/), que incluye una [corrección de seguridad](https://snikket.org/blog/snikket-jan-2021-security-release/) anunciada anteriormente en Enero. La nueva característica principal en este lanzamiento es la funcionalidad para importar/exportar cuentas, la parte final del [proyecto de portabilidad de cuentas XMPP](https://docs.modernxmpp.org/projects/portability/) fundado por NGI DAPSI.
+
+![Proyecto de portabilidad de cuentas XMPP](/images/newsletter/january2022/snikket-account-import-export.png "Proyecto de portabilidad de cuentas XMPP")
+
+## Bibliotecas
+
+Un [nuevo componente XMPP](https://gitlab.com/navlost.eu/xmpp/components/webhooks) fue publicado y podría tener algo de feedback. El componente implementa un transporte [webhook](https://en.wikipedia.org/wiki/Webhook) que le permite a los usuarios (la persona que hostiea el componente y cualquiera al que este permita) crear endpoints HTTP para recibir eventos y traducirlos a mensajes XMPP. Los Webhook payloads son procesados por un middleware y las notificaciones XMPP son basadas en plantillas y escritas en [EJS](https://ejs.co/ "Embedded JavaScript templating"). Actualmente viene con integraciones con GitLab y Git puro y tambien puede usarse con Slack como un middleware crudo y sin probar, pero también procesa texto plano y PNG, JPEG y contenido en PDF, que es enviado a los subscriptores como un adjunto via [HTTP File Upload (XEP-0363)](https://xmpp.org/extensions/xep-0363.html). Puedes encontrar el [repositorio principal](https://gitlab.com/navlost.eu/xmpp/components/webhooks) (todavía sin calidad productiva) y también un [servidor de demostración](https://gitlab.com/navlost.eu/xmpp/components/webhooks/-/blob/devel/README.md#demo-server) disponible para pruebas casuales.
+
+### Propuestas
```

Comment:
> I would also recommend to trabslate the headers like:
> 
> ## New (Nuevo)
> Because the states are also official

Oh! oki ... will fix that

> Where is the XEP section header? and the introduction lines
🤔 let me check

Reply:
found the XEP's section header .. I overlooked it 🤦🏻